app/modules/ads/module.js

```diff
@@ -61,6 +61,7 @@ class Ads {
 
     geoEdge.call();
     krux.call();
+    moatYi.call();
```

Comment:
I like the pattern where the service decides whether is should run or not (instead of each app).